homeassistant/components/technove/config_flow.py

```diff
@@ -0,0 +1,55 @@
+"""Config flow for TechnoVE."""
+
+from typing import Any
+
+from technove import Station as TechnoVEStation, TechnoVE, TechnoVEConnectionError
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_IP_ADDRESS
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+
+class TechnoVEConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for TechnoVE."""
+
+    VERSION = 1
+    discovered_host: str
+    discovered_station: TechnoVEStation
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle a flow initiated by the user."""
+        errors = {}
+        if user_input is not None:
```

Comment:
```suggestion
        if not user_input:
```

Reply:
Changing this as suggested gives me errors like this:
```
error: Value of type "dict[str, Any] | None" is not indexable
```

Reply:
The more you know. Feel free to ignore then

Reply:
I did it the wrong way lol
```suggestion
        if user_input:
```